wallet/wallet.go

```diff
@@ -595,8 +595,12 @@ func (w *Wallet) loadActiveAddrs(ctx context.Context, dbtx walletdb.ReadTx, nb N
 		extn := minUint32(props.LastReturnedExternalIndex+gapLimit, hdkeychain.HardenedKeyStart-1)
 		intn := minUint32(props.LastReturnedInternalIndex+gapLimit, hdkeychain.HardenedKeyStart-1)
 		// pre-cache the pubkey results so concurrent access does not race.
-		extKey.ECPubKey()
-		intKey.ECPubKey()
+		if _, err = extKey.ECPubKey(); err != nil {
```

Comment:
not necessary to change this. the call is made only to prevent a race by precalculating the pubkey bytes.

Reply:
reverted.